configs/approval.json

```diff
@@ -113,9 +131,41 @@
           "resourceDefinitions": [
             {
               "attributeFilter": {
-                "value": "*",
-                "key": "id",
-                "operation": "equal"
+                "key": "scope",
+                "operation": "equal",
+                "value": "admin"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Approval User",
```

Comment:
@hsong-rh Are we missing
"platform_default": true,

For Approval User

Reply:
Good catch!

Reply:
Added! Thanks!